examples/rbac/rstudio-launcher-rbac-0.2.22.yaml

```diff
@@ -0,0 +1,88 @@
+---
```

Comment:
What is the nature of this example? Why is it part of this PR?

Reply:
This is auto generated by one of the jobs when I changed the version of `rstudio-launcher-rbac`.  I don't see the harm in including it, but i also don't know why it exists.

Reply:
no harm, didn't realize it was auto-generated. wacky